db/dcrpg/internal/addrstmts.go

```diff
@@ -155,13 +160,13 @@ const (
 			addresses.value,
 			transactions.block_height,
 			addresses.block_time,
-			tx_vin_vout_index,
-			pkscript
+			addresses.tx_vin_vout_index,
+			vouts.pkscript
 		FROM addresses
 		JOIN transactions ON
 			addresses.tx_hash = transactions.tx_hash
-		JOIN vouts ON addresses.tx_hash = vouts.tx_hash AND addresses.tx_vin_vout_index=vouts.tx_index
-		WHERE addresses.address=$1 AND addresses.is_funding = TRUE AND addresses.matching_tx_hash = '' AND valid_mainchain = TRUE
+		JOIN vouts ON addresses.tx_vin_vout_row_id = vouts.id
```

Comment:
There was no reason to join on tx_hash AND tx_index when `tx_vin_vout_row_id` is the primary key (`id`) in the `vouts` table when `is_funding=true` (and it is in the WHERE below).